app/services/jurisdiction/user_inviter.rb

```diff
@@ -18,13 +18,17 @@ def call
   def invite_users
     users_params.each do |user_params|
       user = User.where.not(role: :submitter).find_by(email: user_params[:email].strip)
-      if user.present? && !user.discarded? && user.confirmed? && !user.regional_review_manager?
+      is_regional_rm = user&.regional_review_manager? || (user && user_params[:role].to_sym == :regional_review_manager)
+
+      if user.present? && !user.discarded? && user.confirmed? && !is_regional_rm
         self.results[:email_taken] << user
-      elsif user&.regional_review_manager? && jurisdiction_id = user_params[:jurisdiction_id]
+      elsif user && is_regional_rm && jurisdiction_id = user_params[:jurisdiction_id]
```

Comment:
`jurisdiction_id = user_params[:jurisdiction_id]` is this meant to be an assignment?